tests/preprocessing/test_triplegs.py

```diff
@@ -287,7 +353,7 @@ def test_only_staypoints_in_trip(self):
         spts_tpls = pd.DataFrame(spts_tpls)
         spts = spts_tpls[spts_tpls["type"] == "staypoint"]
         tpls = spts_tpls[spts_tpls["type"] == "tripleg"]
-        spts_, tpls_, trips = generate_trips(spts, tpls)
+        spts_, tpls_, trips = generate_trips(spts, tpls, add_geometry=False)
```

Comment:
The optional `add_geometry` actually simplified some tests; here for example we don't need to drop the geometry afterwards.